examples/WiFiWebClientWPA2Enterprise/WiFiWebClientWPA2Enterprise.ino

```diff
@@ -0,0 +1,134 @@
+
+/*
+  Web client
+
+ This sketch connects to a website (http://www.google.com)
+ using the WiFi module.
+
+ This example is written for a network using WPA encryption. For
+ WEP or WPA, change the Wifi.begin() call accordingly.
+
```

Comment:
These three lines are repeats of the three lines above
```suggestion
```

Reply:
Please do me a favor and sign your examples when you write them. There are many examples attributed to me which I never wrote, because they were copied from copies of examples I wrote for WiFi libraries years ago. I get questions from students asking me why I wrote something a particular way, when I didn't write it. While I appreciate the acknowledgement, I would like it better if each author credits themselves when they make a new example. If you think someone else deserves credit in the example, write "by YY, based on an older example by XX" if you want. I do that with examples I modify from the rest of you.

Reply:
Absolutely, you are right, amending immediately :wink: